assets/prometheus-k8s/secret-etcd-certs.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
This file was actually depended on by `pkg/manifests/manifests.go` but we were removing it with the `hack/build-jsonnet.sh` script